blog/2022-10-25-new-precipitation-intensity-units.md

```diff
@@ -0,0 +1,13 @@
+---
+author: epenet
+authorURL: https://github.com/epenet
+title: "Add new precipitation intensity units"
+---
+
+As of Home Assistant Core 2022.11, the following constants are deprecated, replaced by `UnitOfVolumetricFlux` enum:
```

Comment:
Let's make it clear that this is only for cases it is about volumetric flux, like precipitation intensity (and not `speed`).